beater/route_config.go

```diff
@@ -79,37 +81,52 @@ var V1Routes = map[string]v1Route{
 
 var V2Routes = map[string]v2Route{
 	V2BackendURL: v2BackendRoute,
-	V2RumURL:     {rumRouteType},
+	V2RumURL:     v2RumRoute,
 }
 
-var v2BackendRoute = v2Route{
-	routeType{
-		v2backendHandler,
-		systemMetadataDecoder,
-		func(*Config) transform.Config { return transform.Config{} },
-	},
-}
+var (
+	v2BackendRoute = v2Route{
+		routeType{
+			v2backendHandler,
+			systemMetadataDecoder,
+			func(*Config) transform.Config { return transform.Config{} },
+			func(c *Config) *lru.Cache { return nil },
```

Comment:
this function is defined 5 times. Could we allow `nil` here to make things simpler? alternatively, we could make a function, call it `noCache` or similar that returns nil and reuse it

Reply:
I'll change it to a nil function 👍